src/main/java/io/apicurio/datamodels/core/validation/rules/invalid/format/AaiInvalidChannelPathRule.java

```diff
@@ -26,6 +27,8 @@
  */
 public class AaiInvalidChannelPathRule extends ValidationRule {
 
+    private static final String RFC_6570_URI_TEMPLATE = "^([^\\x00-\\x20\\x7f\"'%<>\\\\^`{|}]|%[0-9A-Fa-f]{2}|{[+#./;?&=,!@|]?((\\w|%[0-9A-Fa-f]{2})(\\.?(\\w|%[0-9A-Fa-f]{2}))*(:[1-9]\\d{0,3}|\\*)?)(,((\\w|%[0-9A-Fa-f]{2})(\\.?(\\w|%[0-9A-Fa-f]{2}))*(:[1-9]\\d{0,3}|\\*)?))*})*$";
```

Comment:
@EricWittmann is possible to include a dependency to this library https://github.com/hal4j/uritemplate to implement the RFC 6570 validation ??

Reply:
Unfortunately, no - the apicurio-data-models library needs to be zero-dependency.  We just need to do our best with respect to these validation regexps.